data/core.yaml

```diff
@@ -2301,6 +2301,9 @@ en:
         last: "Jump to last node"
         parent: "Select parent way"
         change_parent: "Switch parent way"
+      way_selected:
+        title: "With way selected"
+        child: "Select child node"
```

Comment:
I think this is a lot clearer. Maybe it should say “Select child nodes”, plural, to set the expectation that normally (always?) it would result in many nodes being selected.

Reply:
Let me provide more context and then I would like to hear your opinion:
Method acts in following manner:
1) If its done after doing selectParent, it will return the node from which it started (what was so far toggle '|')
2) If no selectParent was done, then there are 2 cases:
- If single selection -> Will return all nodes
- If multiple selection -> Wil return only common nodes among selected candidates

It's hard to wrap this functionality into a user-friendly short name :)
I wanted to keep it simple and keep the 'antonym'-kind naming with 'Select parent way'.

However if we wanted to be more precise I would recommend:
' Select common child nodes'

Reply:
Actually, I think your suggestion is perhaps the best from user perspective and here is why: Majority of use-cases and most likely first usage, of this feature will probably be case 2.single (abovementioned).  So, going with: 'Select child node**s**'